src/main/java/Jooh/Jooh.java

```diff
@@ -0,0 +1,126 @@
+package Jooh;
+
+import Jooh.ui.Ui;
+import Jooh.storage.Storage;
+import Jooh.parser.Parser;
+import Jooh.task.Task;
+import Jooh.task.TaskList;
+import Jooh.task.Todo;
+import Jooh.task.Deadline;
+import Jooh.task.Event;
+import Jooh.exception.JoohException;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Scanner;
+// Code follows SE-EDU Java coding standards
+
+/**
+ * Entry point for the Jooh chatbot application.
+ * Initializes the user interface, storage, and task list,
+ * then runs an interactive loop to process user commands until exit.
+ */
+public class Jooh {
+
+    /**
+     * Launches the Jooh chatbot.
+     * Sets up storage, loads previously saved tasks, and enters
+     * a command-processing loop that responds to user input.
+     *
+     * @param args Command-line arguments (not used).
+     */
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        TaskList taskList = new TaskList();
+        Storage storage = new Storage();
+        Ui ui = new Ui();
+        ui.welcomeMsg();
+
+        try {
+            List<Task> loaded = storage.load();
+            loaded.forEach(taskList::addTask);
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+
+        while (true) {
+            String raw = sc.nextLine();
+            try {
+                Parser.Parsed p = Parser.Parsed.parse(raw);
+                switch (p.type) {
+                    case BYE: {
+                        ui.goodbyeMsg();
+                        sc.close();
+                        return;
+                    }
```

Comment:
Maybe you could remove the indentation for the case clauses?
```suggestion
                case BYE: {
                    ui.goodbyeMsg();
                    sc.close();
                    return;
                }
```